crypto/src/lib.rs

```diff
@@ -516,14 +591,8 @@ impl<'de> Deserialize<'de> for PublicKey {
 
 impl Decode for PublicKey {
```

Comment:
I believe that you can now derive `Decode`, `Deserialize` and maybe even `Serialize` for `PublicKey`/`PrivateKey`/`KeyPair`